test/functional/feature_minchainwork.py

```diff
@@ -23,7 +23,6 @@
 
 # 2 hashes required per regtest block (with no difficulty adjustment)
 REGTEST_WORK_PER_BLOCK = 2
-
```

Comment:
nit: Inadvertent newline removal?

Reply:
yeah, I could've sworn a previous version introduced this newline and this was me reverting it, but I suppose it must be a rebase artifact, thanks for spotting and fixed now.